CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
+
+## Fork the upstream repository
+
+See https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+## Local changes
+
+Clone the forked repository.
+```bash
+$ git clone git@github.com:<username>/cryostatio.github.io.git
```

Comment:
update references to cryostat instead of cryostatio

the below worked for me:
git clone https://github.com/<username>/cryostat.git

Reply:
Since we are using our `cryostatio.github.io.git` as an example, do you think we should keep as current?

Reply:
> Since we are using our `cryostatio.github.io.git` as an example, do you think we should keep as current?

I think maybe I could find a PR in `cryostat` and replace the screenshots too. That would be easier since cryostat is the main.